packager/media/formats/mp4/mp4_muxer.cc

```diff
@@ -513,6 +517,14 @@ bool MP4Muxer::GenerateAudioTrak(const AudioStreamInfo* audio_info,
     case kCodecOpus:
       audio.dops.opus_identification_header = audio_info->codec_config();
       break;
+    case kCodecMha1:
+      audio.codec_configuration.data = audio_info->codec_config();
+      audio.mhac.extra_data = audio_info->codec_config();
+      break;
+    case kCodecMhm1:
+      //audio.codec_configuration.data = audio_info->codec_config();
+      LOG(WARNING) << "Mux for mhm1" ;
```

Comment:
Does Mhm1 need to be handled differently to Mha1?

Reply:
We've never actually had MHM1 content to test with.